.github/workflows/publish-to-npm.yml

```diff
@@ -0,0 +1,34 @@
+name: publish-to-npm
+run-name: ${{ github.actor }} is publishing to npm
+on:
+  workflow_dispatch:
+    inputs:
+      bump_level:
+        description: type of version bump
+        type: choice
+        default: patch
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+          - custom (use custom_bump_level)
+      custom_bump_level:
```

Comment:
does this always prompt for a `custom_bump_level` even when using patch/minor/major?

Reply:
yes -- there isn't a way to make conditional parameters.